.github/workflows/CI.yaml

```
@@ -155,21 +155,21 @@ jobs:
       ANDROID_NDK_ROOT: /tmp/${{ matrix.NDK_VERSION }}
     steps:    
     - name: Add msbuild to PATH
-      uses: microsoft/setup-msbuild@v1.0.2
-      if: matrix.os == 'windows-2019'
+      uses: microsoft/setup-msbuild@v2
+      if: matrix.os == 'windows-2022'
     - uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)